apps/next-next-app/src/pages/index.tsx

```diff
@@ -0,0 +1,171 @@
+import Image from 'next/image';
+import { Inter } from 'next/font/google';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export default function Home() {
```

Comment:
Please add doc comments to all the Components and functions in the code. All of them

Reply:
added